### DISCUSSION TITLE
Docker Run Error: could not load environment

### DISCUSSION THREAD

**Discussion by EuZak** [DannyDainton/newman-reporter-htmlextra#368] [Q&A] [answered]
Hello! I am new to using docker and can't run reporter-htmlextra with docker.
I use wsl2, Ubuntu 20.04 in Win10.
I can get the report without using docker.
If use docker:
```
$ docker run -t -v $(pwd):/etc/newman dannydainton/htmlextra run /home/test/test.postman_collection.json -e /home/test/test_coll_env.json -r htmlextra
```
```
error: could not load environment
ENOENT: no such file or directory, open '/home/test/test_coll_env.json'
```

**Comment by DannyDainton** [comment, marked as answer]
Can you provide more visual examples of what you can see, the command only leads me to ask questions about the structure of your file system and does that directory contain that file. 

Does it run without the Environment file? 

```
$ docker run -t -v $(pwd):/etc/newman dannydainton/htmlextra run /home/test/test.postman_collection.json -r htmlextra,cli
```
Or using a URL link to the Collection?

```
$ docker run -t -v $(pwd):/etc/newman dannydainton/htmlextra run https://www.getpostman.com/collections/87e7fdca175d74ff3beb -r htmlextra,cli
```
Without more contextual information there's isn't a lot I can do here.

**Reply by EuZak** [reply]
I use local environment file and local collection file

**Reply by DannyDainton** [reply]
Does this run for you?

```
docker run -t -v $(pwd):/etc/newman dannydainton/htmlextra run https://www.getpostman.com/collections/87e7fdca175d74ff3beb -r htmlextra,cli
```

What is your local file structure? What is `/home/test/`? 

Have you tried it without the path, if you're running it in that directory?

$ docker run -t -v $(pwd):/etc/newman dannydainton/htmlextra run test.postman_collection.json -e test_coll_env.json -r htmlextra

Show me what you can see in the terminal.

**Reply by EuZak** [reply]
> Does this run for you?
Yes
![ubuntu2004_Zhnp9LNr9T](https://user-images.githubusercontent.com/98324672/155341732-7eb641d8-03a6-4e3e-beb8-7fed95c965c0.png)

**Reply by EuZak** [reply]
> What is your local file structure? What is `/home/test/`?
> 
> Have you tried it without the path, if you're running it in that directory?
Ubuntu 20.04 in Win10

**Reply by EuZak** [reply]
> $ docker run -t -v $(pwd):/etc/newman dannydainton/htmlextra run test.postman_collection.json -e test_coll_env.json -r htmlextra
> 
> Show me what you can see in the terminal.

OMG. Its magic) 
Everything worked

**Comment by EuZak** [comment]
Thank you!
